Makefile

```diff
@@ -116,6 +116,21 @@ clean:
 
 .PHONY: go-mod-cache clean
 
+FIND_ARGS := -name '*.go' -type f -not -name '*.pb.go'
+
+## format: Run gofmt and goimports.
+format:
+	@echo Formatting...
+	@find . $(FIND_ARGS) | xargs gofmt -d -s
+	@find . $(FIND_ARGS) | xargs goimports -w -local github.com/tendermint/fundraising
```

Comment:
What commands do we expect for MacOS devs to use for the target?

Reply:
I think the shell command itself will be fine.  I'm using `find` and `xargs` in a way that is compatible across both OS.

Reply:
Oh, it was not working in my MacOS but it is working fine now.